Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM mhart/alpine-node:8.9.4
+
+WORKDIR /opt/central-settlement
+COPY config /opt/central-settlement/config
+COPY data /opt/central-settlement/data
+COPY handlers /opt/central-settlement/handlers
+COPY tests /opt/central-settlement/tests
+COPY package.json /opt/central-settlement
+COPY server.js /opt/central-settlement
+COPY settlementAPI.json /opt/central-settlement
```

Comment:
@bothadeon @vgenev is this required? There is another swagger.json in the config folder. Please confirm?